__tests__/integration.test.js

```diff
@@ -77,4 +77,36 @@ describe("/api/articles", () => {
         expect(body.message).toBe("Article not found");
       });
   });
+  test("GET 400 - Incorrect article id type responds with bad request error", () => {
```

Comment:
I'd update this description to be Invalid article ID, and the one before this to be valid but non-existent article ID